spring-cloud-stream-schema/src/main/java/org/springframework/cloud/stream/schema/avro/AvroSchemaMessageConverter.java

```diff
@@ -43,28 +43,31 @@
 	/**
 	 * Create a {@link AvroSchemaMessageConverter}. Uses the default {@link MimeType} of
 	 * {@code "application/avro"}.
+	 * @param manager for schema management
 	 */
-	public AvroSchemaMessageConverter() {
-		super(new MimeType("application", "avro"));
```

Comment:
We cannot remove a publicly available constructor that is already part of the API between minor releases. However, you can add a `deprecation` tag on it and we can remove it when the next major release occurs.

Reply:
added back method as suggested